src/test/java/collab/SearchTest.java

```diff
@@ -21,86 +21,68 @@ public void setUp() {
     empDB.add(new Employee(Arrays.asList("18115040","TTETHU HBO","CL3","010-4581-2050","20080718", "ADV")));
     
     answer = new Employee(Arrays.asList("17112609","FB NTAWR","CL4","010-5645-6122","19860903","PRO"));
-    noAnswer = new Employee(Arrays.asList("19000000","TEST TEST","CL2","010-0000-0000","19000101","EX"));
+    noAnswer = new Employee(Arrays.asList("19010101","TEST TEST","CL2","010-0000-0000","19000101","EX"));
   }
   
-  public int searchTest_EmployeeNum(Employee answer) {
-    List<Employee> empList = empDB.searchItems("employeeNum", answer.getEmployeeNumber());
+  @Test
+  public void searchTestFound() {
+    searchTest_EmployeeNum(true,answer);
+    searchTest_Name(true,answer);
+    searchTest_Cl(true,answer);
+    searchTest_PhoneNum(true,answer);
+    searchTest_BirthDay(true,answer);
+    searchTest_Certi(true,answer);
+  }
+  
+  @Test
+  public void searchTestNotFound() {
+    searchTest_EmployeeNum(false,noAnswer);
+    searchTest_Name(false,noAnswer);
+    searchTest_Cl(false,noAnswer);
+    searchTest_PhoneNum(false,noAnswer);
+    searchTest_BirthDay(false,noAnswer);
+    searchTest_Certi(false,noAnswer);
+  }
+  
+  public void searchTest_EmployeeNum(boolean isMatch, Employee answer) {
+    List<Employee> foundList = empDB.searchItems("employeeNum", answer.getEmployeeNumber());
 
-    if(empList.size()==1) {
-      assertEquals(empList.get(0).getEmployeeNumber(), answer.getEmployeeNumber());
-      return 1;
-    }
-    return 0;
+    if(isMatch) assertEquals(foundList.get(0).getEmployeeNumber(), answer.getEmployeeNumber());
```

Comment:
아마도 answer 부분이 기대값인 것 같아, 두 값의 자리를 바꾸는 것이 어떨까요?

```suggestion
    if(isMatch) assertEquals(answer.getEmployeeNumber(), foundList.get(0).getEmployeeNumber());
```